README.md

```diff
@@ -284,3 +286,5 @@ Contributions and suggestions are very welcome. Check out the guidelines and [ma
 [@uniformfutures]: https://github.com/uniform-futures
 [@h0r0man]: https://github.com/h0r0man
 [@nickwarner]: https://twitter.com/nickwarner
+[@dietrich]: https://twitter.com/dietrich
+[@andgokevin]: https://twitter.com/andgokevin
```

Comment:
I prefer having no author to keep the README more readable, but in the end my mindset is not to be too meticulous about the README to save time. i just merge, if there're tiny things to clean up, i can quickly patch.